build.gradle

```diff
@@ -58,6 +62,8 @@ subprojects {
         }
     }
 
+    project.plugins.apply('net.ltgt.errorprone')
```

Comment:
@artem-zinnatullin I added this. I also saw that you pushed a new branch for the application issue but did not create a PR for that. Any reason for that?